capstone-frontend/src/components/AssignmentCard.jsx

```diff
@@ -44,16 +44,25 @@ class AssignmentCard extends Component {
         });
   }
 
+  onComplete = () => {
+    //TODO #90: Create functionality to mark assignments as complete
+  }
+
   componentDidMount() {
     this.fetchComments();
   }
 
   render() {
     return (
-      <div className="assignment-border">
-        <div> {this.props.assignment.text} </div>
-        <div> Created: {(new Date(this.props.assignment.whenCreated)).toString()} </div> 
-        <div> Due: {(new Date(this.props.assignment.whenDue)).toString()} </div>
+      <div className="assignment-card">
+        <div className="assignment-head">
+          <div>
+            <div className="assignment-date"> Posted: {(new Date(this.props.assignment.whenCreated)).toString()} </div> 
+            <div className="assignment-date"> Due: {(new Date(this.props.assignment.whenDue)).toString()} </div>
+            <div className="assignment-text"> {this.props.assignment.text} </div>
+          </div>
+          <Button onClick={this.onComplete} variant="success"> Mark as Done </Button> 
+        </div> 
```

Comment:
Is this overlap from your other PR?

Reply:
It is. I'm going to base the PR off of the `styling` branch for now to make reviewing easier. Once PR #91 is merged (styling branch) then I'll base off master.